src/stories/TableWithInlineEditing.stories.js

```diff
@@ -70,19 +69,33 @@ storiesOf('TableWithInlineEditing', module)
             <div>{story()}</div>
         </ThemeProvider>
     ))
-    .add('allowed to edit', () => (
+    .add('allowed to edit, add and delete', () => (
+        <TableWithInlineEditing
+            columnsInfo={columnsInfo}
+            content={content}
+            updateContent={{}}
+            allowedToEdit
```

Comment:
updateContent should be a function.  You can just do 

`updateContent={() => {}} `

or similiar

Reply:
Ahh yeah. Now yours is through I'll merge the fixes in to this stuff, will make sure this stuff is in the merged version